libs/@guardian/libs/src/loadScript/loadScript.test.ts

```diff
@@ -75,6 +75,12 @@ describe('loadScript', () => {
 	});
 
 	it('rejects if the script fails to load', async () => {
-		await expect(loadScript(badURL)).rejects.toMatch(badURL);
+		expect.assertions(2);
+		try {
+			await loadScript(badURL);
+		} catch (error) {
+			expect(error instanceof Error).toEqual(true);
+			expect((error as Error).message).toEqual('Error loading script: bad-url');
+		}
```

Comment:
i think this makes it a breaking change? any code that's expecting a string if it fails will break?

Reply:
Ah yes you're right I'll change it to a major release